client/src/components/todolist/TodoList.tsx

```diff
@@ -0,0 +1,62 @@
+import { graphql } from "babel-plugin-relay/macro";
+import { useFragment } from "react-relay";
+import { TodoList_tasks$key } from "./__generated__/TodoList_tasks.graphql";
+import "./Style.css";
+
+interface Props {
+  className: string;
+  tasks: TodoList_tasks$key;
+}
+
+const TodoList = (props: Props) => {
+  const data = useFragment(
+    graphql`
+      fragment TodoList_tasks on Query {
+        allTodoCreateTables {
+          nodes {
+            id
+            completed
+            task
+          }
```

Comment:
this will definitely work for returning the set of nodes (each todo in the database). But when returning a set, I'd suggest using `edges`.
`edges` contain more information about where each result connects within the set and can be used to add / remove "todos" easily using the connection directive.
https://relay.dev/docs/guided-tour/list-data/updating-connections/#adding-edges
```suggestion
         edges {
           node {
              id
              completed
              task
            }
          }
```